plugin-server/src/main/ingestion-queues/session-recording/utils.ts

```diff
@@ -214,30 +214,12 @@ export const parseKafkaMessage = async (
     }
     // end of deprecated mechanism
 
-    const invalidEvents: any[] = []
     const events: RRWebEvent[] = $snapshot_items.filter((event: any) => {
-        if (!event || !event.timestamp) {
-            invalidEvents.push(event)
-            return false
-        }
-        return true
+        // we sometimes see events that are null
+        // there will always be some unexpected data but, we should try to filter out the worst of it
+        return event && event.timestamp
```

Comment:
agree with the change, just wondering if the `timestamp` is either null or invalid generated on our end or is it provided by the rrweb lib? Should we open a new issue in there then?
Maybe related https://github.com/rrweb-io/rrweb/issues/898